src/functional_test_runner/cli.js

```diff
@@ -75,7 +75,6 @@ const functionalTestRunner = createFunctionalTestRunner({
       bail: cmd.bail,
       grep: cmd.grep,
       invert: cmd.invert,
-      require: `ts-node/register --project tests/tsconfig.json -r tsconfig-paths/register -T "test/**/*.{ts,js}"`
```

Comment:
how are you running typescript tests? Couldn't find any solution about running mocha. I was changed this line before since our `src/functional_test_runner/cli.js` file wrapped mocha and launched them with parameters.

Reply:
To run mocha tests you can use `node scripts/mocha`, https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#debugging-unit-tests

Reply:
yeah I mean what about functional tests, what solution are you using for running under typescript without compiling

Reply:
The mocha tests are run in process, which already has ts-node setup by loading `src/setup_node_env`: https://github.com/elastic/kibana/blob/75384404e8f2607247ff1ee4c9f2e57412ae04e1/scripts/functional_test_runner.js#L20-L21

Reply:
Basically, we take care of setting up a unified auto-transpilation environment including babel and typescript via `src/setup_node_env` and always load that as soon as we start any node process. This includes the process that runs the functional test runner thanks to use of `scripts/functional_test_runner`.